central/start-glowroot.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+# 
+
+java $GLOWROOT_OPTS -jar glowroot-central.jar
```

Comment:
`exec java ...` so that it keeps PID 1 and java will receive signals, e.g. from `docker stop ...`

Reply:
Updating it